package.yaml

```diff
@@ -12,7 +12,7 @@ github: "hyperreal-gopher/waffle"
 
 dependencies:
   - base >= 4.7 && < 5
-  - brick
+  - brick == 0.52.1
```

Comment:
I think this is not necessary/is ignored. It grabs 0.52.1 from the resolver I specified.